Northstar.Client/mod/scripts/vscripts/weapons/sh_titancore_utility.gnut

```diff
@@ -0,0 +1,647 @@
+untyped
```

Comment:
This entire file should probably be moved to ``Northstar.Custom``, since its a shared script